msf4j/msf4jspring/.gitignore

```diff
@@ -0,0 +1,25 @@
+/target/
```

Comment:
I'm guessing these might have been added by start.spring.io? You should be able to remove them all, relying on the .gitignore in the parent.